zip-1014.rst

```diff
@@ -248,7 +257,7 @@ Direct-grant option
 It may be deemed better, operationally or legally, if the Major Grant funds
 are not accepted and disbursed by ZF, but rather directly assigned to the
 grantees. Thus, the following mechanism MAY be used in perpetuity for some
-or all grantees, if agreed upon by both ECC and ZF before Network Upgrade 4
+or all grantees, if agreed upon by both BO and ZF before Network Upgrade 4
```

Comment:
I think this should still be `ECC`, as IIRC it's ECC that is party to the trademark agreement, not ECC's owner(s).

Reply:
This was reverted.